test/claim.test.ts

```diff
@@ -114,34 +134,91 @@ describe('StakingPool.claim', () => {
     });
   });
 
-  context('claim after another round initiated', async () => {
-    beforeEach('init second round', async () => {
-      await testEnv.stakingPool.connect(alice).stake(amount);
+  context('pool 1 and pool 2 is opened', async () => {
```

Comment:
Can you explain what is the purpose of these test cases? 
Because there's no `stake` action before user claims, so below checks must be always passed.